src/libifconfig.c

```diff
@@ -401,6 +401,36 @@ ifconfig_get_capability(ifconfig_handle_t *h, const char *name,
 	return (0);
 }
 
+int ifconfig_get_groups(ifconfig_handle_t *h, const char *name,
+    struct ifgroupreq *ifgr)
+{
+	int s, len;
+
+	if (ifconfig_socket(h, AF_LOCAL, &s) != 0) {
```

Comment:
style(9) asks us to be consistent about braces for single line if/while/... blocks.